crates/re_space_view_time_series/src/visualizer_system.rs

```diff
@@ -190,6 +206,19 @@ impl TimeSeriesSystem {
                     .annotation_info();
                 let default_color = DefaultColor::EntityPath(&data_result.entity_path);
 
+                let override_color = lookup_override::<Color>(data_result, ctx).map(|c| {
+                    let arr = c.to_array();
+                    egui::Color32::from_rgba_unmultiplied(arr[0], arr[1], arr[2], arr[3])
+                });
+
+                let override_label =
+                    lookup_override::<Text>(data_result, ctx).map(|t| t.to_string());
+
+                let override_scattered =
+                    lookup_override::<ScalarScattering>(data_result, ctx).map(|s| s.0);
+
+                let override_radius = lookup_override::<Radius>(data_result, ctx).map(|r| r.0);
```

Comment:
Yeah, it's not awful, esp since we restrict things to splats at the moment, but I'd still love to have it automagically happen inside one of the archetype-view-style generic queries.